lib/send_text.rb

```diff
@@ -0,0 +1,41 @@
+require 'twilio-ruby'
+require 'envyable'
+Envyable.load('./config/env.yml', 'development')
+
+class SendText
+
+  attr_reader :customers
+
+  def confirmation_text
+    sender
+  end
+
+  private
+
+    def sender
+
+      accountSID = ENV[accountSID]
+      authToken = ENV[authToken]
+
+      @client = Twilio::REST::Client.new accountSID, authToken
+
+      from = ENV[usernumber]
+
+      @customers = { :"+44712345" => :Chris }
+
+      @customers.each do |key, value|
+        message = @client.account.messages.create(
+          :from => from,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)